packages/parrot-devtools/__tests__/app/components/__snapshots__/DevTools.spec.jsx.snap

```diff
@@ -62,9 +63,10 @@ exports[`DevTools component renders settings 1`] = `
       src={null}
     />
     <button
-      className="btn-icon btn-inline btn-tertiary dls-icon-setting"
+      className="btn-icon btn-inline btn-primary dls-icon-setting"
       onClick={[Function]}
-      title="Show settings"
+      title="Hide settings"
+      type="button"
     />
   </styled.div>
 </styled.div>
```

Comment:
Did you find a way to create a dev build of the Chrome Extension and test it with these changes?

Reply:
No I haven't.. I can address this in a different PR